OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -147,17 +473,246 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <date notBefore="1800" notAfter="1974"/>
+                                <seg type="ink">Black.</seg>
+                                <desc>An untrained hand.</desc>
+                            </handNote>
+                            
+                            <handNote xml:id="h5" script="Ethiopic" corresp="#a6">
+                                <locus from="47r" to="51v"/>
+                                <date notBefore="1800" notAfter="2006"/>
+                                <seg type="ink">Black.</seg>
+                                <desc>An untrained, irregular hand.</desc>
                             </handNote>
                         </handDesc>
                         
+                        <additions>
+                            <list>
+                                <item xml:id="a1" corresp="#h2">
+                                    <locus from="1r" to="2v"/>
+                                    <desc type="MagicText">Magic prayer containing excerpts from the Psalms of David. The name or names mentioned at the end of the text have
+                                        been erased, and that of <persName ref="PRS13177HaylG"/>, hardly legible, has been added in a crude hand above the erasure.</desc>
+                                    <q xml:lang="gez">
+                                        <seg part="I">በስመ፡ አብ፡ ወወልድ<gap reason="illegible"/>ቅዱስ፡ ፩ አምላክ፡
+                                            አስማ<gap reason="illegible"/>ቅድስት፡ መግረሬ፡ ፀር፡ <gap reason="illegible"/>እግዚአብሔር፡
+                                            ወይዘ<gap reason="illegible"/>ይጉየዩ፡ ጸላእት፡ ከማሁ፡ <gap reason="illegible"/>ሉ፡ 
+                                            ኃጥአን፡ እምቅድመ፡ <gap reason="illegible"/>ዚአብሔር፡ ወከማሁ፡ <gap reason="illegible"/>ሙ፡
+                                        ለርጉማን፡ ሳጥናኤል፡</seg>
+                                        <seg part="F">በኃይለ፡ መስቀሉ፡ ለንጉሠ፡ ሰማያት፡ ወምድር፡ በዘአጥመቀከ፡ አባ፡ ዮሐንስ፡ በደምከ፡ ክቡር፡ ወበሥጋከ፡ ኅብስተ፡ ሕይወት፡
+                                        ወበደምከ፡ ጽዋዓ፡ መድኃኒት፡ እምሕላክሙ፡ በ፭ቅንዋተ፡ መስቀል፡ እለ፡ ቀነውዎ፡ ውስተ፡ ሥጋከ፡ በዲበ፡ ዕፀ፡ መስቀል፡ 
+                                        ክቡር፡ ሳዶር፡ አላዶር፡ ዲናት፡ አዲ<gap reason="illegible"/>፡ ሮ<pb n="2v"/>ዳስ፡ <seg type="supplication">በዝንቱ፡ አስማቲከ፡
+                                        አድኅኖ፡ ለገብርከ፡ 
+                                            <del rend="erasure"/><add place="overstrike"><persName ref="PRS13177HaylG"><hi rend="rubric"><sic resp="DR">ኃይል ጌዮጌስ</sic> ጊዮርጊስ</hi></persName></add></seg></seg>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a2" corresp="#h3">
+                                    <locus from="3r" to="4r"/>
+                                    <desc type="MagicText">Magic prayer to protect against criminals, containing <foreign xml:lang="gez">ʾasmāt</foreign> and Amharic words.</desc>
```

Comment:
Would it be useful to create a general work record for "magic prayer" or better protectice prayer against criminals"?